credentialAtomicQueryMTPWithRelay.go

```diff
@@ -1,223 +1,218 @@
+	UserAuthClaimNonRevMtp      []string         `json:"userAuthClaimNonRevMtp"`
+	UserAuthClaimNonRevMtpAuxHi *merkletree.Hash `json:"userAuthClaimNonRevMtpAuxHi"`
+	UserAuthClaimNonRevMtpAuxHv *merkletree.Hash `json:"userAuthClaimNonRevMtpAuxHv"`
+	UserAuthClaimNonRevMtpNoAux string           `json:"userAuthClaimNonRevMtpNoAux"`
+	UserClaimsTreeRoot          *merkletree.Hash `json:"userClaimsTreeRoot"`
+	//UserState                   *merkletree.Hash `json:"userState"`
+	UserRevTreeRoot   *merkletree.Hash `json:"userRevTreeRoot"`
+	UserRootsTreeRoot *merkletree.Hash `json:"userRootsTreeRoot"`
+	UserID            string           `json:"userID"`
+
+	Challenge             string `json:"challenge"`
+	ChallengeSignatureR8X string `json:"challengeSignatureR8x"`
+	ChallengeSignatureR8Y string `json:"challengeSignatureR8y"`
+	ChallengeSignatureS   string `json:"challengeSignatureS"`
+
+	IssuerClaim                     *core.Claim      `json:"issuerClaim"`
+	IssuerClaimClaimsTreeRoot       *merkletree.Hash `json:"issuerClaimClaimsTreeRoot"`
+	IssuerClaimIdenState            *merkletree.Hash `json:"issuerClaimIdenState"`
+	IssuerClaimMtp                  []string         `json:"issuerClaimMtp"`
+	IssuerClaimRevTreeRoot          *merkletree.Hash `json:"issuerClaimRevTreeRoot"`
+	IssuerClaimRootsTreeRoot        *merkletree.Hash `json:"issuerClaimRootsTreeRoot"`
+	IssuerClaimNonRevClaimsTreeRoot *merkletree.Hash `json:"issuerClaimNonRevClaimsTreeRoot"`
+	IssuerClaimNonRevRevTreeRoot    *merkletree.Hash `json:"issuerClaimNonRevRevTreeRoot"`
+	IssuerClaimNonRevRootsTreeRoot  *merkletree.Hash `json:"issuerClaimNonRevRootsTreeRoot"`
+	IssuerClaimNonRevState          *merkletree.Hash `json:"issuerClaimNonRevState"`
+	IssuerClaimNonRevMtp            []string         `json:"issuerClaimNonRevMtp"`
+	IssuerClaimNonRevMtpAuxHi       *merkletree.Hash `json:"issuerClaimNonRevMtpAuxHi"`
+	IssuerClaimNonRevMtpAuxHv       *merkletree.Hash `json:"issuerClaimNonRevMtpAuxHv"`
+	IssuerClaimNonRevMtpNoAux       string           `json:"issuerClaimNonRevMtpNoAux"`
+	ClaimSchema                     string           `json:"claimSchema"`
+	IssuerID                        string           `json:"issuerID"`
+	Operator                        int              `json:"operator"`
+	SlotIndex                       int              `json:"slotIndex"`
+	Timestamp                       int64            `json:"timestamp,string"`
+	Value                           []string         `json:"value"`
+
+	RelayProofValidClaimsTreeRoot *merkletree.Hash `json:"relayProofValidClaimsTreeRoot"`
+	RelayProofValidRevTreeRoot    *merkletree.Hash `json:"relayProofValidRevTreeRoot"`
+	RelayProofValidRootsTreeRoot  *merkletree.Hash `json:"relayProofValidRootsTreeRoot"`
+	RelayState                    *merkletree.Hash `json:"relayState"`
+	UserStateInRelayClaim         *core.Claim      `json:"userStateInRelayClaim"`
+	UserStateInRelayClaimMtp      []string         `json:"userStateInRelayClaimMtp"`
```

Comment:
JFI, I don't see a proof of non-revocation for UserStateInRelayClaim

Reply:
Yes. Do not check revocation for now as it is not clear how we implement relay switch for identity